modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -2445,11 +2445,13 @@ private void updateScrollBarsAndCells(boolean recreate) {
             }
             else {
                 if (isVertical) {
-                    hbar.resizeRelocate(0, (viewportLength-hbar.getHeight()),
-                            viewportBreadth, hbar.prefHeight(viewportBreadth));
+                    double prefHeight = hbar.prefHeight(viewportBreadth);
+                    hbar.resizeRelocate(0, viewportLength-prefHeight,
```

Comment:
Minor : Code formatting convention: Add spaces around '-' at 4 places in your changes.
It was not followed in earlier version, but let's correct it while touching these lines.

Reply:
Yes, makes sense, I thought about doing it while removing the unnecessary parenthesis but didn't want to be too intrusive. Done.